### ISSUE TITLE
Does import ignore attachments?

### ISSUE THREAD

**Issue by jacobischwartz** [wp-cli/wp-cli#2326]
Hello,

I've just started using wp-cli and tried it out on a multisite install, to import a specific site. The import runs well, but I notice that none of the media attachments are imported. When importing without the CLI (via webpage), there's a checkbox to import images and attachments. Is there an equivalent flag that I should be adding to the import?

Thanks,

Jacob

**Comment by danielbachhuber** [comment]
> The import runs well, but I notice that none of the media attachments are imported.

WP-CLI imports attachments by default. If they aren't getting imported, there's likely something incorrect about your export file.

**Comment by jacobischwartz** [comment]
Hi Daniel,

Good to know. I was able to get it working.

FYI - I had tried the import previously through the web interface, which had imported all the files before pooping out. So I used the CLI to empty the site before trying an import. I think the issue might have been that I still had the uploaded files in the uploads directory before running the CLI import. I guess if the files are already there, the attachments aren't created...? When I cleared out the files and did a fresh import, the attachments were created fine.

Thanks,

Jacob
